vnext/Desktop.IntegrationTests/WebSocketServer.cpp

```diff
@@ -148,9 +148,9 @@ std::shared_ptr<BaseWebSocketSession<boost::asio::ip::tcp::socket>> WebSocketSes
   return this->shared_from_this();
 }
 
-#pragma endregion // BaseWebSocketSession
```

Comment:
>// B [](start = 18, length = 4)

(nit) it would be nice if all mechanical changes were part of a separate PR, that way it would be easier to review the meat of the code

Reply:
The overall purpose of the PR was to format and document WebSocket-related types.
Because actual functionality doesn't change, I considered these changes easy to gloss over.

Reply:
Please split into separate code reviews in the future, it will be easier on all code reviewers.

---
In reply to: [298413375](https://github.com/microsoft/react-native-windows/pull/2662#discussion_r298413375) [](ancestors = 298413375)